components/FoodCard.js

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+import Link from "next/link";
```

Comment:
I think you do not need to import the `Link` component as you do not use it in this file 🧹 

```suggestion

```